pkg/pipeline/bean/CiBuildConfig.go

```diff
@@ -28,15 +28,24 @@ type CiBuildConfigBean struct {
 }
 
 type DockerBuildConfig struct {
-	DockerfilePath     string            `json:"dockerfileRelativePath,omitempty"`
-	DockerfileContent  string            `json:"dockerfileContent"`
-	Args               map[string]string `json:"args,omitempty"`
-	TargetPlatform     string            `json:"targetPlatform,omitempty"`
-	Language           string            `json:"language,omitempty"`
-	LanguageFramework  string            `json:"languageFramework,omitempty"`
-	DockerBuildOptions map[string]string `json:"dockerBuildOptions,omitempty"`
-	BuildContext       string            `json:"buildContext,omitempty"`
-	UseBuildx          bool              `json:"useBuildx"`
+	DockerfilePath         string                   `json:"dockerfileRelativePath,omitempty"`
+	DockerfileContent      string                   `json:"dockerfileContent"`
+	Args                   map[string]string        `json:"args,omitempty"`
+	TargetPlatform         string                   `json:"targetPlatform,omitempty"`
+	Language               string                   `json:"language,omitempty"`
+	LanguageFramework      string                   `json:"languageFramework,omitempty"`
+	DockerBuildOptions     map[string]string        `json:"dockerBuildOptions,omitempty"`
+	BuildContext           string                   `json:"buildContext,omitempty"`
+	UseBuildx              bool                     `json:"useBuildx"`
+	UseBuildxK8sDriver     bool                     `json:"useBuildxK8sDriver,omitempty"`
+	BuildxK8sDriverOptions []BuildxK8sDriverOptions `json:"buildxK8SDriverOptions,omitempty"`
+}
+
+type BuildxK8sDriverOptions struct {
```

Comment:
can we make this generic map, so that if anyone wants to pass new Param, can add that ??